lib/CollectionsTable/styles.scss

```diff
@@ -48,31 +49,64 @@ $collections-table-indent-level-support: 10;
 .collections-table__cell {
   border-top: $collections-table-border;
   border-bottom: $collections-table-border;
-
+  padding: ($collections-table-content-padding + 1) $collections-table-content-padding;
+  .collections-table__row--selected & {
+    border-top: $collections-table-selected-border;
+    border-bottom: $collections-table-selected-border;
+    padding: $collections-table-content-padding;
+  }
   &:first-child {
     padding: 0;
     border: 0;
   }
 
   &:first-child > .collections-table__cell-content {
-    padding: $collections-table-spacing;
+    padding: ($collections-table-spacing + 1) $collections-table-spacing ($collections-table-spacing + 1) ($collections-table-spacing + 1);
```

Comment:
Why do we need the + 1px?

Reply:
so it doesnt "jiggle" when adding the extra 1px border